android/samples/westerosemployees-hybrid/src/main/java/com/liferay/mobile/screens/westerosemployees_hybrid/views/DocsCard.java

```diff
@@ -0,0 +1,109 @@
+package com.liferay.mobile.screens.westerosemployees_hybrid.views;
+
+import android.content.Context;
+import android.os.Handler;
+import android.os.Looper;
+import android.util.AttributeSet;
+import android.view.ViewPropertyAnimator;
+
+import com.liferay.mobile.screens.asset.AssetEntry;
+import com.liferay.mobile.screens.portlet.PortletConfiguration;
+import com.liferay.mobile.screens.portlet.PortletDisplayListener;
+import com.liferay.mobile.screens.portlet.PortletDisplayScreenlet;
+import com.liferay.mobile.screens.portlet.util.InjectableScript;
+import com.liferay.mobile.screens.westerosemployees_hybrid.R;
+import com.liferay.mobile.screens.westerosemployees_hybrid.utils.CardState;
+
+/**
+ * @author Víctor Galán Grande
+ */
+public class DocsCard extends CommentsRatingsCard implements PortletDisplayListener {
```

Comment:
Have you talked about using westeros as a dependency to not have to copy all these classes over again?